internal/commands/resolve.go

```diff
@@ -86,8 +113,19 @@ func ResolveIncidentByDialog(ctx context.Context, client bot.Client, logger log.
 		return err
 	}
 
-	channelAttachment := createResolveChannelAttachment(incident, userName)
-	privateAttachment := createResolvePrivateAttachment(incident)
+	isPostMortemMeeting, err := strconv.ParseBool(postMortemMeeting)
+	if err != nil {
+		return err
+	}
+
+	if isPostMortemMeeting {
+		//postMortemMeetingURL = calendar.CreateCalendarEvent() [locked - card do feijo ]aqui vai ser o tratamento de erros.
+		calendar.CreateCalendarEvent()
+		postMortemMeetingURL = "link ficticio"
+	}
```

Comment:
In our PR we are not returning a string here, but a struct that contains information about the PM meeting. One of the fields is the URL.